requirements.txt

```diff
@@ -20,3 +20,4 @@ simplejson==3.17.0
 boto3==1.12.28
 seaborn==0.10.0
 click==7.1.1
+pydantic==1.0.0
```

Comment:
I think this should be 1.4 (weird naming on their part)

Reply:
done